conf/makecheck-sle-15.1.yaml

```diff
@@ -1,5 +1,6 @@
 flavor: b2-30
-image: makecheck-sle-15.1-x86_64
+#image: makecheck-sle-15.1-x86_64
+image: minimal-sle-15-sp1.x86_64-openstack
```

Comment:
Wrong name after https://gitlab.suse.de/ceph/storage-ci/-/merge_requests/116 gets merged.

Reply:
That's fine, feel free to commit corresponding change, I was working with the images I had.